README.md

```diff
@@ -1,2 +1,45 @@
 # terraform-aws-s3-cloudfront-site
```

Comment:
I think that this should be renamed to no longer reference CloudFront, but also to specify that we're trying to create an origin S3 bucket for a CloudFront distribution, not necessarily a web site (we aren't making use of the S3 website functionality).